templates/403.html

```diff
@@ -4,7 +4,11 @@
     <meta charset="UTF-8">
     <title>403</title>
     <script type="text/javascript">
-    window.location = "https://code.org/educate";
+        {% if request.user.is_staff %}
+            window.location = "/admin";
+        {% else %}
+            window.location = "https://code.org/educate";
```

Comment:
I didn't find a way to test with a non-staff user, since non-staff users already get redirected to login on any admin page. so this else clause is a somewhat speculative measure to keep non-staff users (perhaps if we ever fix the "site" option on the login page) from getting stuck in a redirect loop.